sig/members.rbs

```diff
@@ -185,6 +185,8 @@ module RBS
         include _HashEqual
 
         def update: (?name: Symbol, ?type: Types::t, ?ivar_name: Symbol | false | nil, ?kind: kind, ?annotations: Array[Annotation], ?location: loc?, ?comment: Comment?, ?visibility: visibility?) -> instance
+
+        def ivar: () -> InstanceVariable?
```

Comment:
Does it still make sense for `ivar_name` to accept false in the constructor and the attr_reader?

Reply:
Good question, since there's backwards compatibility concerns there.

We can add a new keyword arg (perhaps `explicit_ivar_name:`?) and use that, while still supporting the old.